frontend_tests/node_tests/stream_list.js

```diff
@@ -30,6 +30,7 @@ const scroll_util = mock_esm("../../static/js/scroll_util", {
 mock_esm("../../static/js/ui", {get_scroll_element: ($element) => $element});
 mock_esm("../../static/js/unread", {
     num_unread_for_stream: () => num_unread_for_stream,
+    stream_has_any_unread_mentions: () => stream_has_any_unread_mentions,
```

Comment:
I'm going to merge this, but I think it'd be better to change this test to use the real `unread.js` module; it's a data module, so it should be easy to setup with a bit of data for the relevant streams rather than mocking it.